src/mare/compiler/target.cr

```diff
@@ -0,0 +1,9 @@
+# TODO: Bring in "crystal/src/compiler/crystal/codegen/target.cr"
```

Comment:
Is it possible to just `require "compiler/crystal/codegen/target"` and use that class verbatim?